docs/sources/mimir/manage/mimir-runbooks/_index.md

```diff
@@ -639,6 +639,27 @@ How to **investigate**:
       - If you encounter any Compactor resource issues, add CPU/Memory as needed temporarily, then scale back later.
       - You can also optionally scale replicas and shards further to split the work up into even smaller pieces until the situation has recovered.
 
+### MimirCompactorHasRunOutOfDiskSpace
+
+This alert fires when the compactor has run out of disk space at least once.
+When this happens the compaction will fail and after some time the compactor will retry the failed compaction, but it is very likely that on each retry it will just hit the same disk space limit again and it won't be able to recover on its own.
```

Comment:
This is true when using compactor concurrency = 1. With higher value, it's all about what combination of jobs is running at the same time, and it's much trickier to argue that same condition happens again.

This is also reason why we prefer to use concurrency=1 and scale compactors horizontally.

Reply:
How about this: https://github.com/grafana/mimir/pull/8278/commits/8bb0ed1a03c6a42cdcacd45c5d446c1b1c1adb8d